src/components/Home/LastStarredRepo.js

```diff
@@ -0,0 +1,71 @@
+'use client';
```

Comment:
I need to put some error handling on this in case I forget to regenerate my github auth token again 💀